src/main/java/com/github/vkorobkov/jfixtures/loader/FixtureValue.java

```diff
@@ -30,12 +30,12 @@ public FixtureValue(Object value) {
 
     @Override
     public String toString() {
-        return String.valueOf(value);
+        return value == null ? "NULL" : String.valueOf(value);
```

Comment:
:+1:

Reply:
Maybe we can create a private method
```java
private String getSqlRepresentation() {
 ....
}
```

and just to call it from `toString()` so it gives a hint why we have this check and upper cased `NULL`

Reply:
More, I think that `toString()` is abused here. Probably we need to have a public method `getSqlRepresentation` and use it in places when we generate SQL.

`toString`, though, can return a normal object representation: `type` and `value` fields. Possibly we can just generate `toString` with lombok. Thoughts?